src/main/java/com/mycompany/app/App.java

```diff
@@ -48,12 +50,18 @@ public void start(Stage stage) {
         Scene scene = new Scene(stackPane);
         stage.setScene(scene);
 
+        // An API key is required to enable access to services, web maps, and web scenes hosted in ArcGIS Online.
```

Comment:
I think we still need that standard line in here about it not being good practice to store the API key in source code.